src/utilities/getThresholds.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Returns an array of thresholds to build the curve.
+ * @param {Array} prediction Array of predictions.
+ * @return {Array} An array of thresholds.
+ */
+
+export function getThresholds(prediction: number[]) {
+  let uniques = [...new Set(prediction)].sort((a, b) => a - b);
```

Comment:
Quite interesting syntax. I did many tests and it is really the fastest way. In this case typed array are very slow using Chrome (but not in Safari). Seems there should be still some improvements in v8 to do there.

https://jsbench.me/w8kryhfull/1